NEWS.md

```diff
@@ -1,5 +1,6 @@
 # devtools 1.8.0
-
+* New `use_cran_badge()` adds a CRAN status badge that you can copy into a README file. Green indicates package is on CRAN. Packages not yet submitted or accepted to CRAN get a red badge.
```

Comment:
Btw. archived packages are also red. 

Btw.2. Obviously, if someone else has an package with the same name on CRAN, then that package will be on the badge. Do we want a note about this in the docs?